src/components/dato/PeriodeDatovelger.tsx

```diff
@@ -77,9 +80,16 @@ const PeriodeDatovelgere: FC<Props> = ({
 
   return (
     <>
-      <FeltGruppe classname={'utenlandsopphold__spørsmål'}>
+      <div className={'utenlandsopphold__spørsmål'}>
```

Comment:
utenlandsopphold__spørsmål?

Reply:
Gjenbruker css fra medlemskap. 
Burde kanskje endre på navnet her ja 🙈 
Kanskje heller bruke styled components for specificity's sake 🤔